src/renderer/screens/DatabaseScreen/ExportModal/index.tsx

```diff
@@ -213,6 +222,18 @@ export default function ExportModal({ data, onClose }: ExportModalProps) {
             console.error(e);
             setStage('ERROR');
           });
+      } else if (format === "json") {
```

Comment:
We can follow this for JSON

https://github.com/querymx/querym/blob/main/src/renderer/screens/DatabaseScreen/QueryResultViewer/useDataTableContextMenu.tsx#L81

We just need to do

```
jsonString = // can follow the above code;
window.electron.saveTextFile(fileName, jsonString);
```